mlir/lib/Conversion/AIRRtToNpuPass.cpp

```diff
@@ -1371,7 +1371,7 @@ struct AIRRtToNpuPass : public impl::AIRRtToNpuBase<AIRRtToNpuPass> {
         // configure shim tile
         if (chanToIdMap.count(dstColIndex) == 0)
           chanToIdMap[dstColIndex] = 15;
-        int bdID = chanToIdMap[dstColIndex];
+        int &bdID = chanToIdMap[dstColIndex];
```

Comment:
Maybe `chanToIdMap[dstColIndex]--` would be more clear to future readers of the code?

Reply:
> Maybe `chanToIdMap[dstColIndex]--` would be more clear to future readers of the code?

fixed in https://github.com/Xilinx/mlir-air/pull/596